radiodns/config/urls.py

```diff
@@ -34,11 +34,12 @@
     path("clients/", include("apps.clients.urls")),
     path("radiovis/", include("apps.radiovis.urls")),
     path("radioepg/", include("apps.radioepg.urls")),
+    path("radiodns/", include("apps.si_pi.urls")),
     path("system/", include("apps.system.urls")),
     path("admin/", admin.site.urls),
 ]
 #if settings.DEBUG:
-#    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
the if debug is there for a reason. In the prod/staging we want nginx to be serving files.